app/services/pull_request.rb

```diff
@@ -4,7 +4,7 @@ class PullRequest
               :url, :body, :language, :repo_name
 
   def initialize(params)
-    @info = params.dig(:pull_request)
```

Comment:
trying to avoid nil errors

Reply:
Params never gonna be nil so why this can raise an error? 🤔

Reply:
I didn't know anything about the Slack connection/gem/client/etc usages (I still don't).
I just started to test the class in isolation and noticed that it was raising a lot errors because it assumes a complex parameter tree structure. I usually try to validate initialization/params just a bit, to avoid ugly bugs later in the code (most of the time, `undefined xxxx for nil` errors.

Reply:
sometimes a complex input structure is a smell that could be improve/cleaned, sometimes with separated params or a Struct/PORO